.github/workflows/golang_lint_and_test.yml

```diff
@@ -9,6 +9,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
+        with:
+            go-version: '1.17'
```

Comment:
We do not want to peg the version of Go.  We are not production code, so we do not need reproducable builds.  Like other TheAlgorithms repos, we want to be running the latest and greatest so that we are alerted when a new version of the language breaks our existing algorithms.

Reply:
I strongly agree!